src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfigurationTest.java

```diff
@@ -60,9 +60,12 @@ public class KinesisClientLibConfigurationTest {
                                            .withAccountId(ACCOUNT_ID)
                                            .withResource("stream/" + TEST_STRING)
                                            .build();
+
+    // Invalid steamARN due to invalid service. This is a sample used for testing.
+    // See testWithInvalidStreamArnsThrowsException for more examples
```

Comment:
1. Member variable documentation should be Javadoc for visibility (e.g., Javadoc tools, IDE tooltips). Inline comments are better reserved for when you expect eyeballs on _that_ specific line (e.g., reading the internals of a method body).
2. `@see` may be used which can provide a navigable Javadoc link.

Reply:
Fixed to be a javadoc comment